pkg/opsmanager/api_get_backup_configs.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opsmanager
+
+import (
+	"encoding/json"
+
+	"github.com/mongodb-labs/pcgc/pkg/httpclient"
+	"github.com/mongodb-labs/pcgc/pkg/useful"
+)
+
+// BackupConfigs response structure
+type BackupConfigs struct {
+	Links         []Link         `json:"links"`
+	BackupResults []BackupResult `json:"results"`
+	TotalCount    int            `json:"totalCount"`
+}
+
+// BackupResult is backup result))
+type BackupResult struct {
```

Comment:
We're missing `IncludedNamespaces` here. We should at least expose it, even if we don't use it right now.